knowledge_doc_converter/knowledge_doc_converter/tasks/conversion_task.py

```
@@ -0,0 +1,315 @@
+            resp = callback_client.notify_started(
+                path=callback_status_path,
+                document_id=document_id,
+                generation=index_generation,
+            )
+            if not resp.get("ok") or not resp.get("document_exists"):
+                logger.info(
+                    f"[Conversion] Skipped: document_id={document_id}, " f"resp={resp}"
+                )
+                record_conversion_skipped("not_exists_or_stale")
+                return {"status": "skipped", "reason": "not_exists_or_stale"}
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)